### ISSUE TITLE
[#123] [Shattered Halls] existential dread kills players in 

### ISSUE THREAD

**Issue by kelno** [ValorenWoW/sunstrider-issues#444]
<a href="https://github.com/plutonium447"><img src="https://avatars0.githubusercontent.com/u/40911527?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [plutonium447](https://github.com/plutonium447)**
_Friday Dec 28, 2018 at 05:51 GMT_
_Originally opened as https://github.com/Nolin7777/TBC-Bugtracker/issues/123_

----

**Current behavior:**
Players advancing through the start of Shattered Halls realise all their failures are their own responsibility and that their personal internal failures are a result of unchangeable human nature so that they are doomed to repeat failure again and again until they die. then they die.

![wowscrnshot_122818_153952](https://user-images.githubusercontent.com/40911527/50504290-4f2b2a80-0ab8-11e9-9669-e7e187689843.jpg)

![wowscrnshot_122818_154009](https://user-images.githubusercontent.com/40911527/50504265-1723e780-0ab8-11e9-873d-2375a97c2bfc.jpg)

tl;dr
some invisible supermob/effect is nuking players in SH

**Expected behavior:**
Players should live.

**Steps to recreate issue**
Step 1: Enter Shattered Halls
Step 2: clear first room
step 3: walk up hallway
step 4: experience personal hell

**id's:**

**Source(s)**
https://www.wowdb.com

**Comment by kelno** [comment]
Fixed
